heat/core/tests/test_io.py

```diff
@@ -147,6 +148,10 @@ def test_load_csv(self):
         with self.assertRaises(TypeError):
             ht.load_csv(self.CSV_PATH, header_lines="3", sep=";", split=0)
 
+    @unittest.skipIf(
+        len(TestCase.get_hostnames()) > 1,
+        "Test only works on single node, file creation is not synchronized across nodes",
```

Comment:
```suggestion
        len(TestCase.get_hostnames()) > 1 or not os.environment.get('TMPDIR'),
        "Test only works on single node, file creation is not synchronized across nodes. Set the environment variable 'TMPDIR' to a path globally accessible by all nodes otherwise",
```

Reply:
Could this cause problems when TMPDIR is set only on some of the nodes? Can that happen? I'm also not sure if this guarantees that TMPDIR is globally accessible by all the nodes.

Reply:
I guess if it is meant for us, it should be good enough. I don't think there are a lot of people out there running the tests.